website/docs/docs/cloud/connect-data-platform/connect-starburst-trino.md

```diff
@@ -4,18 +4,18 @@ description: "Configure Starburst/Trino connection."
 sidebar_label: "Connect Starburst/Trino"
 ---
 
-The following are the required fields for setting up a connection with Starburst Enterprise, Starburst Galaxy, and Trino cluster. Unless specified, "cluster" will mean any of these products' clusters.     
+The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), or [Trino](https://trino.io/) cluster. Unless specified, "cluster" means any of these products' clusters.
```

Comment:
going off of past experience, future tense is ok in this situation since it is referring to what the person is going to encounter. 

```suggestion
The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), and [Trino](https://trino.io/) cluster. Unless specified, "cluster" will mean any of these products' clusters.
```

Reply:
That typically goes against style guides and I would prefer if we can avoid future tense. Also specifically in this context the future tense is wrong ... the meaning is in the docs right now.

Reply:
I've supported several style guides and have found from experience that it depends. Some of those experiences included specialized technical editors, too. totally a non-blocker for me! 🚀